src/main.ts

```diff
@@ -3,8 +3,32 @@ import * as envalid from 'envalid';
 import { app, CommitArgs, PullRequestArgs } from './app';
 import { ActionUtils } from './utils/action-utils';
 
+const commitArgFields = ['paths', 'message', 'token', 'amend'];
+
+const pullRequestArgFields = [
+  'title',
+  'body',
+  'base',
+  'labels',
+  'assignees',
+  'reviewers',
+  'teamReviewers',
```

Comment:
```suggestion
  'team-reviewers',
```

Add a test for this fix.